packages/Pagination/src/Pagination.tsx

```diff
@@ -50,12 +50,14 @@ const Pagination: React.FC<PaginationProps> = (
     defaultItemsPerPage: itemsPerPage
   });
 
+  const pages = !totalItems ? null : calculatePages();
+
+  if (pages && pages.length === 1) return null;
```

Comment:
is it possible for there to be 0 pages? should the check be `pages.length <= 1`?